.buildkite/pipeline.yml

```diff
@@ -24,6 +24,7 @@ steps:
       - "julia --project -e 'using Pkg; Pkg.precompile()'"
       - "julia --project -e 'using Pkg; Pkg.status()'"
 
+      - "julia --project=experiments -e 'using Pkg; Pkg.develop(path = \".\")'"
```

Comment:
Hm, we have a manifest checked in for experiments and that has the path specified: https://github.com/CliMA/ClimaLSM.jl/blob/ad31773009c9309d5e45b083603873a8914be286/experiments/Manifest.toml#L215

so I *think* we should be ok without this? not a problem to add in if needed, but I wonder why

Reply:
Ah - noted. I'll check the Manifest files in.